spec/unit/type/selboolean_spec.rb

```diff
@@ -32,6 +32,22 @@
     Puppet::Type.type(:selboolean).new(:name => "yay", :value => :off)
   end
 
+  it "should support :true as a value to :value" do
+    Puppet::Type.type(:selboolean).new(:name => "yay", :value => :true)
+  end
+
+  it "should support :false as a value to :value" do
+    Puppet::Type.type(:selboolean).new(:name => "yay", :value => :false)
+  end
+
+  it "should support true as a value to :value" do
+    Puppet::Type.type(:selboolean).new(:name => "yay", :value => true)
+  end
+
+  it "should support false as a value to :value" do
+    Puppet::Type.type(:selboolean).new(:name => "yay", :value => false)
+  end
+
```

Comment:
This is more about how these tests are written in general and not this PR specifically, but when you have cases with repeating code like here, I like to do:

```ruby
[true, :true, false, :false].each do |val|
  it "should support |val| as a value to :value" do
    Puppet::Type.type(:selboolean).new(:name => "yay", :value => val)
  end
end
```